tests/manager/manager-executeCommand-001.phpt

```diff
@@ -54,7 +54,7 @@ object(MongoDB\Driver\Cursor)#%d (%d) {
   ["readPreference"]=>
   NULL
   ["isDead"]=>
-  bool(true)
+  bool(false)
```

Comment:
Per https://github.com/mongodb/mongo-php-driver/pull/322/files#r63595485, the cursor is no longer dead because we've yet to iterate to the first and only result document of the command.